### PR TITLE
Rdb improvements

### PR DESCRIPTION
**Pull request by triwav**
See commit messages for a list of what all what was done as a part of this PR

https://user-images.githubusercontent.com/1753881/177899012-e9fbad03-d9d2-469a-8614-783f32b1cd16.mov

.

### CODE REVIEW

File: ui/rdb/components/SceneGraphInspectorPanel/NodeBranchView.svelte [hunk 1]
```diff
@@ -51,10 +55,125 @@
         dispatch('open', nodeTree);
     }
 
+    let nodeInfoHideTimeout: ReturnType<typeof setTimeout>;
+    function startNodeInfoTimeout() {
+        stopNodeInfoTimeout();
+        nodeInfoHideTimeout = setTimeout(async () => {
+            nodeInfoElement.classList.add('hide');
```
Comment (TwitchBronBron) [inline ui/rdb/components/SceneGraphInspectorPanel/NodeBranchView.svelte:62]: I think we ran into issues doing manual css class modifications in the past with svelte. Shouldn't this really be toggling a `.visible` property on the data, and let svelte show/hide it?

File: ui/rdb/components/SceneGraphInspectorPanel/NodeDetailView.svelte [hunk 2]
```diff
@@ -130,6 +126,62 @@
...
+    }
+
+    function onResetValueButtonClicked() {
+        const elements = [];
+        for (const child of this.parentElement.children) {
+            if (child.classList.contains('fieldValue')) {
+                elements.push(child);
+            };
+        }
+        if (elements.length === 1) {
```
Comment (TwitchBronBron) [inline ui/rdb/components/SceneGraphInspectorPanel/NodeDetailView.svelte:164]: Please add comments about WHY the length is 1 or 2. 

What happens if length is 0 or > 2? Should those be handled?

**Review by TwitchBronBron — CHANGES_REQUESTED** [review]
Two small things, but overall it looks great!

File: ui/rdb/components/SceneGraphInspectorPanel/NodeBranchView.svelte (hunk 1 above)
Comment (triwav) [reply ui/rdb/components/SceneGraphInspectorPanel/NodeBranchView.svelte]: Updated here. We do use it in the NodeDetailView and it seems to work ok but this one is a simple 1:1 relationship so I switched it

File: ui/rdb/components/SceneGraphInspectorPanel/NodeDetailView.svelte (hunk 2 above)
Comment (triwav) [reply ui/rdb/components/SceneGraphInspectorPanel/NodeDetailView.svelte:166]: Added comments and a console log in case we ever hit it

**Review by TwitchBronBron — APPROVED** [review]

**Pull request merged**